dependencies.sh

```diff
@@ -20,7 +20,9 @@ gpgcheck=1
 gpgkey=https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
 EOF
 
-dnf install -y wget rust coreos-installer kubectl libvirt podman qemu-img qemu-kvm virt-install make git golang-bin virt-viewer
+dnf install -y wget rust coreos-installer kubectl libvirt podman qemu-img qemu-kvm virt-install make git golang-bin virt-viewer osbuild-composer composer-cli cockpit-composer bash-completion firewalld lorax
```

Comment:
added dependencies for microshift in dependencies.sh to keep it consistent

Reply:
I actually prefer your original way. But I will let Balazs weigh in.

Reply:
I think we're good here based on balazs's last comments